docker-app/qfieldcloud/core/models.py

```diff
@@ -691,6 +692,64 @@ def active_users(self, period_since: datetime, period_until: datetime):
             is_staff=False,
         ).filter(Q(id__in=users_with_delta) | Q(id__in=users_with_jobs))
 
+    def list_active_users_jobs_deltas_count(self) -> List[Dict[str, Union[str, int]]]:
+        """Return a List of Dictionaries mapping user ids into the following values:
+        - user_id
+        - username
+        - number of jobs scheduled by the user
+        - number of deltas scheduled by the user
+        """
+        delta = timedelta(weeks=4)
```

Comment:
Is this referring to 'current billing period'. Are you sure this is 4 weeks and not time since last invoice?

Reply:
The success of test added to this PR makes me want to say, 'Yes'.

Reply:
I do not get what you mean with the `success of test added to this PR?` . The test does not assert anything that relates to a timerange or 'last billing period'?

Further, are you sure it is the 'last billing period' and not the current / recent one?

![image](https://github.com/opengisch/qfieldcloud/assets/21113500/8f89999f-3377-4389-8d7b-35c83be71b09)

Reply:
> Maybe we ask Ivan here how to best get the current / last billing period? But maybe look for some existing utility first.
> 
> I will continue in a bit ...
> 
> Looks good sofar :)

If you look at https://github.com/opengisch/qfieldcloud/blob/62c8419ab5359c3b8a7d2eb88a84d881cc8f2961/docker-app/qfieldcloud/core/models.py#L660-L661, does it answer your question?

Reply:
Don't think so. It does not mention billing period anywhere ... ?

Reply:
I'd put it this way: the notion of active users used in the ClickUp task == the notion of active users used in OrganizationAdmin (already defined and used in master) == the notion of active users used in this PR. The only difference is that we need a specific time window. I suggest "the last 4 weeks ", but I have no specific criterion in mind. Is there something you disagree with? Or is there a specific criterion to determine the time window that I am unaware of?

Reply:
@suricactus Perhaps I am missing something about the requirements here. What notion of `active_users` should this PR be using?

Reply:
@why-not-try-calmer I asked in the support channel what they prefer. IMO you can also change the description to `1 month from now` or so. Just needs to be correct. Else it would cause more trouble then help...

Reply:
The added specifications are in the clickup task comment.

Reply:
Okay, so all is fine or do you see something wrong in the current implementation?